bin/atat-web-api.ts

```diff
@@ -31,4 +32,8 @@ if (utils.isPossibleTemporaryEnvironment(environmentId)) {
 // cdk.Aspects.of(app).add(new NIST80053R5Checks({ verbose: true }));
 cdk.Aspects.of(app).add(new GovCloudCompatibilityAspect());
 
-const apiStack = new AtatWebApiStack(app, `${environmentName}AtatWebApiStack`, { environmentName });
+const apiStack = new AtatPipelineStack(app, `Pipeline`, {
```

Comment:
Will we only ever have one stack named `Pipeline` within the account? Maybe?

Reply:
What's the disadvantage of something like `AtatWebApiPipeline` instead?  Poor readability?